desktop-app/src/renderer/components/Previewer/Device/Toolbar.tsx

```diff
@@ -124,14 +126,24 @@ const Toolbar = ({
     setFullScreenshotLoading(false);
   };
 
+  const toggleRulers = async () => {
+    if (webview === null) {
+      return;
+    }
+    toggleRuler();
+  };
+
   const rotate = async () => {
     setRotated(!rotated);
     onRotate(!rotated);
   };
 
   return (
-    <div className="flex items-center justify-between gap-1">
-      <div className="my-1 inline-flex max-w-[75%] items-center gap-1 overflow-x-auto">
+    <div
+      className="flex items-center justify-between gap-1"
+      style={{ paddingLeft: '30px' }}
```

Comment:
Can we not add the`paddingLeft` to the device name row and toolbar row? 
We can only add it for the device row as this adds an unnecessary indentation for the title and the toolbar row. 

Also, can we add the margin to the device only when the ruler is activated? As it doesn't make a lot of sense to have this empty space all other times as well.

Reply:
I tried that but that means the screens are jumping 30px all the time.

Changing position of the view messes with muscle memory.

The only thing we could try is reducing the 30 px to less, but as I also tried that, it makes it less readable. 

I don't think the space is an issue. Only at first. After a day, you wont notice it anymore. 

From a ux point, moving elements all the time is not a good option, layout instability is an annoying issue. If you have a better idea, I am all ears.